outputs.tf

```diff
@@ -171,8 +171,8 @@ output "aws_auth_configmap_yaml" {
   value = templatefile("${path.module}/templates/aws_auth_cm.tpl",
     {
       eks_managed_role_arns                   = [for group in module.eks_managed_node_group : group.iam_role_arn]
-      self_managed_role_arns                  = [for group in module.self_managed_node_group : group.iam_role_arn if group.platform != "windows"]
-      win32_self_managed_role_arns            = [for group in module.self_managed_node_group : group.iam_role_arn if group.platform == "windows"]
+      self_managed_role_arns                  = [for group in module.self_managed_node_group : group.iam_instance_profile_arn if group.platform != "windows"]
```

Comment:
Thank you @bryantbiggs for the quick follow up.  I think iam_instance_profile_arn is the actual the iam instance profile? https://github.com/terraform-aws-modules/terraform-aws-eks/blob/master/modules/self-managed-node-group/main.tf#L149-L151

Reply:
Instance profile is not what is used for access within the cluster, its just a construct used by the EC2 service in order to assume the role. 

This has been like this for quite some time so I am highly confident it is correct as is otherwise we'd have a large number of users complaining that their self-managed node groups fail to join the cluster

Reply:
Thank you @bryantbiggs  for the clarification!  revert this part of change